doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+in your history. Next, authorize Docker to access this repository
+with:
+
+.. code::
+
+    GH_USERNAME=<my-github-username>
+    cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
+
+
+.. _run_an_mapdl_image:
+
+Run an MAPDL Docker image
+=========================
+
+You can now launch MAPDL from Docker using the command line, a
+`docker compose file <run_an_mapdl_image_using_docker_compose_>`_ 
+or a script that gather the commands given to the command line.
+
+Your Docker image should have a valid MAPDL license configuration.
+The easiest way is to have an environment variable :env:`ANSYSLMD_LICENSE_FILE`
+pointing to a valid license server. This environment variable can be already
+included in the Docker image. However this is not recommended because it could
+expose the license server if the Docker image get leaked.
+The recommended approach is to set that environment variable when running the
+container. 
+
+To instantiate an MAPDL Docker container from an image hosted at ``ghcr.io/myuser/myrepo/mymapdldockerimage`` use:
+
+.. code:: bash
+
+  ANSYSLMD_LICENSE_FILE=1055@MY_LICENSE_SERVER_IP
+  LOCAL_MAPDL_PORT=50053
+  MAPDL_DOCKER_REGISTRY_URL=ghcr.io/myuser/myrepo/mymapdldockerimage
+  docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052 $MAPDL_DOCKER_REGISTRY_URL -smp > log.txt
+
+For Windows use:
+
+.. code:: pwsh-session
+
+    $env:ANSYSLMD_LICENSE_FILE="1055@MY_LICENSE_SERVER_IP"
+    $env:LOCAL_MAPDL_PORT=50053
+    $env:MAPDL_DOCKER_REGISTRY_URL="ghcr.io/myuser/myrepo/mymapdldockerimage"
+
+    docker run -e ANSYSLMD_LICENSE_FILE=$env:ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $env:LOCAL_MAPDL_PORT`:50052   $env:MAPDL_DOCKER_REGISTRY_URL -smp
+
+
+First time you run it, Docker logins into the registry and
+pulls the required image.
+This can take some time depending on the size of the image.
+
+To rerun it, you should restart the container:
+
+.. code:: bash
+
+   docker start mapdl
+
+Or just delete it and run it again using:
+
+.. code:: bash
+
+    docker rm -f mapdl
+
+    docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always --name mapdl -p $LOCAL_MAPDL_PORT:50052   $MAPDL_DOCKER_REGISTRY_URL -smp > log.txt
+
+
+You can append the Docker flag ``--rm`` to automatically clean up the container
+when it exits.
+
+The preceding commands create a log file (``log.txt``) in your current directory location, you can
+remove the following part ``> log.txt `` to avoid this. In that case the command
+output is redirected to the console which is kept blocked until the Docker image exits.
+You can detach the console from the Docker container output by appending ``-d`` to the
+`docker run <docker_run_>`_  command (always add this before the Docker image URL).
+If you don't want to block the console, the best approach is to pipe the output to a file
+as mentioned before so you can inspect the output of that file.
```

Comment:
```suggestion
The preceding commands create a log file (``log.txt``) in your current directory location. However, you can
remove ``> log.txt `` if you don't want to create this file. In this case, the command
output is redirected to the console, which is kept blocked until the Docker image exits.
You can detach the console from the Docker container output by appending ``-d`` to the
`docker run <docker_run_>`_  command. (Always add this before the Docker image URL.)
If you don't want to block the console, the best approach is to pipe the output to a file
as mentioned earlier so that you can inspect the output of that file.
```